sagemaker-pipelines/step-decorator/local-mode-example/using_step_decorator_with_local_mode.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p>We can add one more cell to retrieve the execution results, e.g. the evaluation report:</p><pre class="ql-syntax" spellcheck="false">evaluation_report = execution.result(step_name="AbaloneEval")
print(evaluation_report)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>